src/miral/basic_window_manager.cpp

```diff
@@ -288,9 +288,6 @@ void miral::BasicWindowManager::refocus(
     if (hint && select_active_window(hint))
         return;
 
-    if (can_activate_window_for_session_in_workspace(application, workspaces_containing_window))
-        return;
```

Comment:
@AlanGriffiths this was here since you introduced MirAL… 8 years ago…

Can you comment on whether you're OK with this change?

Reply:
It seems like only yesterday!

With what I now know about window management, it just looks wrong.